TabNine.py

```diff
@@ -1,11 +1,49 @@
-from .tab_nine_process import tabnine_proc
-from .lib.settings  import is_native_auto_complete
 import sublime_plugin
 import sublime
+import sys
+
+if int(sublime.version()) >= 3114:
```

Comment:
Can we extract this to a function, and give it a name?

Reply:
done